app/controllers/documents_controller.rb

```diff
@@ -123,9 +123,7 @@ def form_params
   end
 
   def set_planning_application
-    @planning_application = authorize(PlanningApplication.find(
-                                        params[:planning_application_id],
-                                      ))
+    @planning_application = current_local_authority.planning_applications.find(params[:planning_application_id])
```

Comment:
so what we had before with pundit on here, that did the authorise on the planning application find, it would render an explicit error saying not authorised if the planning application find failed. We could do a quick if else to render the 402 not authorised, or at least catch the error? 
I guess I did mention it friday but was just mulling through it again in terms of best practice if we should not always be explicit instead of returning a not found. Hmm

Reply:
It is a 404, as southwark.bops/planning_applications/1 is not the correct URL for that planning application, it is bucks.bops/planning_applications/1

It would be a 403 if the planning application was accessible from the same URL but logged in as a different user.